spec/community.mediawiki

```diff
@@ -55,3 +55,29 @@ of any unfilled orders.
 
 Less drastic punitive measures such as a cool-down period may be considered
 for minor, first-time or infrequent conduct violations.
+
+===Penalization Notification===
+
+In the event that a rule is broken, the client will be sent a notification that
+includes the broken rule, a detailed description of the offense, the time of
```

Comment:
I usually try not to over-describe the structure that is about to be described in the table. For this paragraph, I would drop the list of field descriptions and refer to "penalization details" or something.